src/ray/core_worker/core_worker.cc

```diff
@@ -6,6 +6,10 @@
 
 namespace {
 
+// The max allowed size in bytes of a return object from direct actor calls.
+// Objects larger than this size will be spilled to plasma.
+const int kMaxDirectCallObjectSize = 100000;
```

Comment:
Why not use the same check that we currently do when deciding whether task args should be inlined or not?

Reply:
The inline logic is written in Python unfortunately.